### ISSUE TITLE
Log in: "Required" text needs to be part of the label for required form fields 

### ISSUE THREAD

**Issue by ndolter** [mmiron-HC/WST-RSAMS#129]
URL: https://hcdev.openplus.ca/en/user/login
User Agent: Chrome (v95)
Language: Both

Issue Description: Required form fields "Username" and "Password" with red asterisk (*) when passed through CSS don't have "Required" text in the label.  Additionally, although not a WCAG failure, the placement of the asterisk is typically situated before the label as a best practice for scan ability.

Steps to reproduce:
1.	Access the Log In page via the url above
2.	Select 'Disable All Styles' via the Web Developer tool
3.	There is no indication that the 2 fields are required
<label for="edit-name" class="control-label js-form-required form-required">Username</label>
![image](https://user-images.githubusercontent.com/68117281/140067809-8e8c4575-8b46-4ceb-8edb-3de980adb3bd.png)

Expected Results: Text "Required" should be part of the input label for required form fields that have the asterisk(*) passed through CSS styles.
See: https://www.w3.org/WAI/WCAG21/Techniques/failures/F3

**Comment by echouin** [comment]
Completed in both EN & FR:
﻿﻿https://rsams.prod.cloud.openplus.ca/en/user/login﻿
﻿﻿https://rsams.prod.cloud.openplus.ca/fr/user/login﻿

**Comment by davidgotlieb** [comment]
From OP:
﻿this is Drupal core functionality and has been thoroughly tested.
﻿The field has an attribute of required and aria is set to required. A screen reader can definitely tell that these fields are required.

**Comment by hydegeoff** [comment]
Turning off CSS is something that users' do to get rid some of the fancy features on the page, because they could interfere with the assistive technologies.  It is better to put the word "required" in the field label so for those visual users who turn off CSS.  This also removing some aria to avoid screen reader repetition.

On another note: 
Visual users need to know what the red star represents.  Not all users know a red star means a required field.  Therefore, putting text near the top of the page to explain the red star.

**Comment by echouin** [comment]
I believe that this has been rectified with the new Drupal 10 upgrade? I do not have the tool to test this issue.
